src/ui/AdvancedSearch/DateInput/InTheLastDateInput.ts

```diff
@@ -20,13 +20,13 @@ export class InTheLastDateInput extends DateInput {
 
   public build(): HTMLElement {
     super.build();
-    let input = $$('fieldset', { className: 'coveo-advanced-search-date-input' });
+    const input = $$('fieldset', { className: 'coveo-advanced-search-date-input' });
     (<HTMLFieldSetElement>input.el).disabled = true;
 
-    this.spinner = new NumericSpinner(this.onChange.bind(this));
+    this.spinner = new NumericSpinner(this.onChange.bind(this), undefined, undefined, l('InTheLast'));
```

Comment:
Now that the constructor has 4 arguments, would it not be better to pass an object?

Same for the `Dropdown` constructor on line 29.

Reply:
Would have been a good idea, but unfortunately this is already exported globally and documented, so it would be a breaking change for people relying on those components to change completely the constructor signature.

Adding new optional parameter is not a breaking change on the other hand,